apps/studio/src/server/modules/site/site.service.ts

```diff
@@ -38,7 +38,7 @@ export const setSiteConfig = async (
 ) => {
   return db
     .updateTable("Site")
-    .set({ config })
+    .set({ config: jsonb(config) })
```

Comment:
previously when we didn't cast it as jsonb it still worked right? how does the cast help now?

Reply:
It still worked because of luck. we did not trigger any ambiguity on the pg engine. In more complex shapes, postgres may throw. So this is insurance.